src/clue/app-config.json

```diff
@@ -233,114 +233,42 @@
-      "content": "curriculum/example-curriculum/example-no-group-share.json"
-    },
-    "example-no-my-work": {
-      "content": "curriculum/example-curriculum/example-no-my-work.json"
-    },
-    "example-show-nav-panel": {
-      "content": "curriculum/example-curriculum/example-show-nav-panel.json"
-    },
-    "example-config-subtabs": {
-      "content": "curriculum/example-curriculum/example-config-subtabs.json"
-    },
-    "example-no-section-problem-tab": {
-      "content": "curriculum/example-curriculum/example-no-section-problem-tab.json"
-    },
-    "example-template": {
-      "content": "curriculum/example-curriculum/example-template.json"
-    },
-    "example-variables": {
-      "content": "curriculum/example-curriculum/example-variables.json"
-    },
-    "example-shared-data": {
-      "content": "curriculum/example-curriculum/example-shared-data.json"
-    },
-    "faw": {
-      "content": "curriculum/filling-and-wrapping/filling-and-wrapping.json"
-    },
-    "ffpc": {
-      "content": "curriculum/frogs-fleas-and-painted-cubes/frogs-fleas-and-painted-cubes.json"
-    },
-    "fsss": {
-      "content": "curriculum/flip-spin-slide-stretch/flip-spin-slide-stretch.json"
-    },
-    "ggg": {
-      "content": "curriculum/growing-growing-growing/growing-growing-growing.json"
-    },
-    "hlit": {
-      "content": "curriculum/how-likely-is-it/how-likely-is-it.json"
-    },
-    "help": {
-      "content": "curriculum/help/help.json"
-    },
-    "lfp": {
-      "content": "curriculum/looking-for-pythagoras/looking-for-pythagoras.json"
-    },
-    "m2s": {
-      "content": "curriculum/m2studio/m2studio.json"
-    },
-    "m2s-proto": {
-      "content": "curriculum/m2studio/m2studio-prototype.json"
-    },
-    "moth": {
-      "content": "curriculum/mothed/mothed.json",
-      "guide": "curriculum/mothed/mothed-teacher-guide.json"
-    },
-    "mods": {
-      "content": "curriculum/mods/mods.json",
-      "guide": "curriculum/mods/mods_TG.json"
-    },
-    "msa": {
-      "content": "curriculum/moving-straight-ahead/moving-straight-ahead.json",
-      "guide": "curriculum/moving-straight-ahead/msa-teacher-guide.json"
-    },
-    "pc": {
-      "content": "curriculum/pc2/pc2.json"
-    },
-    "sad": {
-      "content": "curriculum/shapes-and-designs/shapes-and-designs.json"
-    },
-    "sap": {
-      "content": "curriculum/samples-and-populations/samples-and-populations.json"
-    },
-    "sas": {
-      "content": "curriculum/stretching-and-shrinking/stretching-and-shrinking.json",
-      "guide": "curriculum/stretching-and-shrinking/sas-teacher-guide.json"
-    },
-    "sws": {
-      "content": "curriculum/say-it-with-symbols/say-it-with-symbols.json"
-    },
-    "twmm": {
-      "content": "curriculum/thinking-with-math-models/thinking-with-math-models.json"
-    }
-  },
+  "curriculumBaseUrl": "https://models-resources.concord.org/clue-curriculum/branch",
   "unitCodeMap": {
```

Comment:
Instead of adding a new map of legacy directory names to their related unit codes, I thought it'd make sense to use this existing unitCodeMap since it seems to have been set up for somewhat similar reasons.